kolibri/core/assets/src/core_app_constructor.js

```diff
@@ -44,15 +48,22 @@ function Lib() {
  */
 module.exports = function CoreApp() {
   this.lib = new Lib();
+  this.resources = new ResourceManager(this);
   const mediator = new Mediator();
 
+  this.resources.registerResource('contentmetadata', 'id', ContentMetaDataResource);
```

Comment:
Perhaps, for the rare cases where we do need to use something else, that can just be done as an attribute within the `Resource` subclass? Or perhaps we can just punt it for now, until the need arises?